gitcha.go

```diff
@@ -51,6 +51,30 @@ func IsPathInGit(path string) bool {
 // FindFiles finds files from list in path. It respects all .gitignores it finds
 // while traversing paths.
 func FindFiles(path string, list []string) (chan SearchResult, error) {
+	return findFiles(path, list, nil)
+}
+
+// FindFilesExcept finds files from a list in a path, excluding any matches in
+// a given set of ignore patterns. It also respects all .gitignores it finds
+// while traversing paths.
+func FindFilesExcept(path string, list, ignoreList []string) (chan SearchResult, error) {
+	return findFiles(path, list, ignoreList)
+}
+
+func FindFirstFile(path string, list []string) (SearchResult, error) {
+	ch, err := findFiles(path, list, nil)
+	if err != nil {
+		return SearchResult{}, err
+	}
+
+	for v := range ch {
+		return v, nil
+	}
+
+	return SearchResult{}, nil
+}
+
+func findFiles(path string, list, ignorePatterns []string) (chan SearchResult, error) {
```

Comment:
I think we can just rename this function to the publicly exposed `FindFilesExcept` and drop the stub above. `FindFiles` would then still be provided for convenience, directly calling the now public `FindFilesExcept`.

Reply:
Oh, you're absolutely right. I'll update right after lunch 😋